cpp/dolfinx/mesh/graphbuild.cpp

```diff
@@ -174,9 +242,14 @@ compute_nonlocal_dual_graph(
             0};
   }
 
-  const int tdim = mesh::cell_dim(cell_type);
-  const int num_vertices_per_facet
-      = mesh::num_cell_vertices(mesh::cell_entity_type(cell_type, tdim - 1));
+  // Some processes may have empty map
+
+  std::int32_t num_vertices_per_facet;
+  MPI_Allreduce(&facet_cell_map.shape[1], &num_vertices_per_facet, 1,
```

Comment:
The first argument looks unsafe to me - the type is isn't clear. Assign to a variable with the correct type first, then pass address.